lms/static/sass/xblocks/problem_builder.scss

```diff
@@ -0,0 +1,39 @@
+// Overrides for Problem Builder
```

Comment:
If you think it's helpful we can add a link to the original version of this file here: https://github.com/open-craft/problem-builder/blob/7e76920d02c14e3fa2a4deadf8388f52ecb3d873/problem_builder/public/themes/apros.css
So people know where this comes from.

Reply:
👍 addressed with https://github.com/mckinseyacademy/mcka-theme/pull/11/commits/ac5cf98969143434290b94e91820df75e184e977#diff-7f8e5c657cdce1929b229a9122c572feR2